### PR TITLE
Speaker/momchil

### PR DESCRIPTION
**Pull request by tottomotto**

### CODE REVIEW

File: docs/default-firebase-data.json [hunk 1]
```diff
@@ -335,17 +335,20 @@
...
       "title": "Lecture 7"
     },
     "114": {
-      "description": "Lecture 8.",
+      "description": "Challenges in incorporating ML models in corporate business environment",
       "language": "English",
-      "title": "Lecture 8"
+      "title": "Challenges in incorporating ML models in corporate business environment",
+      "speakers": ["momchil"],
+      "tags": ["Machine Learning"]
```
Comment (bstrandjev) [inline docs/default-firebase-data.json:351]: shorten this to ML

**Review by bstrandjev — APPROVED** [review]

File: docs/default-firebase-data.json (hunk 1 above)
Comment (tottomotto) [reply docs/default-firebase-data.json:351]: Any additional tags besides ML?
